sample/SampleAspNetCoreApp/Controllers/HomeController.cs

```diff
@@ -23,25 +23,41 @@ public class HomeController : Controller
 
 		public HomeController(SampleDataContext sampleDataContext) => _sampleDataContext = sampleDataContext;
 
-		public async Task<IActionResult> Index()
+		private bool GetCaptureControllerActionAsSpanFromQueryString()

```

Comment:
Why don't we just use `AddSampleData` for this test? I thought that's done for testing that.

Reply:
The question applies to the `Index` method...

Och.. yeah.. that's HTTP post, and this is easier to test, right? 

But then why do we use `SafeCaptureSpan<IActionResult>` in `AddSampleData`?

Reply:
I've added `SafeCaptureSpan<IActionResult>` to `AddSampleData` at the point when I mistakenly thought that there are no automatic tests for Entity Framework Core so I used `AddSampleData` to verify manually. After you told me that there are automatic Entity Framework Core tests I thought it would be a good idea to add test for the code in this PR affecting Entity Framework Core auto captured spans.

Reply:
I see, yeah.

But that means the `AddSampleData` is not really utilized in this PR, right? ...with the comment I think it's fine to leave it as it is. I just thought we should use the existing method for testing this, but given it's a POST now I also think just adding another one is easier.

Reply:
We can try to use `AddSampleData` for tests as well and yes it will take more effort to simulate form submission but I'm more concerned about DB growing after each test run.

Reply:
> We can try to use AddSampleData for tests as well
I think we are good for now. I don't see it needed within this PR.